### PR TITLE
Fix viewstate race condition

### PR DESCRIPTION
**Pull request by kckst8**

### CODE REVIEW

File: packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx [hunk 1]
```diff
@@ -94,25 +96,69 @@ const Loader: React.FC<ModelLoaderProps> = React.memo(
...
+    }, [
+      connection,
+      viewportOptions,
+      blankConnectionViewState,
+      viewCreatorOptions,
+    ]);
+
+    useEffect(() => {
+      void getViewState();
+    }, [connection, viewportOptions, blankConnectionViewState, getViewState]);
```
Comment (aruniverse) [inline packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx:160]: missing viewCreatorOptions?

Comment (kckst8) [reply packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx]: yep. fixed

**Review by aruniverse — APPROVED** [review]
lgtm, approving, but would like final approval from @StefanRetief before merging

**Review by StefanRetief — APPROVED** [review]
Tested and works

**Pull request merged**